modes/basic-test-mode/src/toolbarButtons.ts

```diff
@@ -649,6 +643,144 @@ const toolbarButtons: Button[] = [
       ],
     },
   },
+
+  // More...
```

Comment:
Seems like we should break this down into two separate files, since this is getting overwhelming